### PR TITLE
Add missing class member initializations found by TypeScript

### PR DESCRIPTION
**Pull request by mohsen1**
**What kind of change does this PR introduce?**
refactor

**Did you add tests for your changes?**
no

**If relevant, link to documentation update:**
n/a

**Summary**

As part of https://github.com/webpack/webpack/pull/6862 I'm adding class member initializers for properties that were added to class instances 

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Comment by mohsen1** [comment]
is that Codacy/PR Quality Review something I should look into?

File: lib/optimize/ChunkModuleIdRangePlugin.js [hunk 1]
```diff
@@ -6,6 +6,7 @@
 class ChunkModuleIdRangePlugin {
 	constructor(options) {
 		this.options = options;
+		this.chunks = undefined;
```
[the remark this answers is not in the document]
Comment (ooflorent) [reply lib/optimize/ChunkModuleIdRangePlugin.js]: Funny. I was doing a PR (#6954) while your comment popped 😅

[the remark this answers is not in the document]
Comment (sokra) [reply lib/optimize/ChunkModuleIdRangePlugin.js]: Great. @mohsen1 you can remove this from your PR.

**Comment by TheLarkInn** [comment]
@mohsen1 since you have a bunch of these smaller PR's do you have any opposition to anyone taking them if they have small corrections? Happy to find some folks to help out here!

**Comment by mohsen1** [comment]
@TheLarkInn I love to get help for this. Having all those initializations in one PR is not ideal. I like to have a pull request per class so we can discuss what's going on and why those initializations are necessary.

**Comment by TheLarkInn** [comment]
That's perfect. Then we can enlist help from the masses knowing this :).

**Comment by sokra** [comment]
Thanks

**Pull request merged**